public/welcome.html

```diff
@@ -0,0 +1,99 @@
+<head>
```

Comment:
I wasn't able to reach this page without manually going to "/welcome.html". We will route to this page in future PRs?